README.md

```diff
@@ -199,7 +207,31 @@ An optional part is using the programming language `Python` and its package
 manager `pip`. To install `Python` and `pip` please refer to the
 [official Python documentation](https://wiki.python.org/moin/BeginnersGuide/Download).
 
-Otherwise, let's define the configuration common to all the instructions.
+Otherwise, let's start in a newly created directory.
+
+First, download the snapshot of the DVC remote and extract it.
+
+```bash
+wget https://zenodo.org/record/4589007/files/bbs_dvc_remote.tar.gz
+tar xf bbs_dvc_remote.tar.gz
+```
+
+Second, clone the Blue Brain Search repository for v0.1.0.
+
+```bash
+git clone --depth 1 --branch v0.1.0 https://github.com/BlueBrain/Search.git
+```
+
+Third, keep track of the path to the working directory, the repository
+directory, and the data and models directory.
```

Comment:
Maybe I could be better to say `keep track of the paths of the working directory, the repository directory, and the data and models directory`. However, it is really a minor comment and I am not confident about what I am suggesting!

Reply:
The sentence is coming from #229 (see [here](https://github.com/BlueBrain/Search/pull/229/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5) lines 220-221).

@Stannislav was the author of this PR. Maybe he knows about what is valid English?

Reply:
Could you have a look at this @Stannislav?

Reply:
To me either way sounds fine :)

Reply:
Lets keep it that way :)